bugbug/repository.py

```diff
@@ -691,10 +703,8 @@ def download_commits(repo_dir, rev_start=0, ret=False, save=True):
     commits_to_ignore = get_commits_to_ignore(repo_dir, commits)
     print(f"{len(commits_to_ignore)} commits to ignore")
 
-    calculate_experiences(commits, commits_to_ignore, first_pushdate, save)
-
-    # Exclude commits to ignore.
-    commits = [commit for commit in commits if commit not in commits_to_ignore]
+    for commit in commits:
+        commit.ignored = commit in commits_to_ignore
```

Comment:
If you're not using commits_to_ignore anymore, maybe you can set this directly in get_commits_to_ignore